static_src/test/unit/actions/user_actions.spec.js

```diff
@@ -168,6 +169,182 @@ describe('userActions', function() {
+    beforeEach(function (done) {
+      userGuid = "fake-udid";
+      user = { guid: userGuid };
+      inviteData = { new_invites:[{ userId: userGuid }] };
+      sandbox.stub(userActions, 'sendUserInviteEmail').returns(Promise.resolve());
+      sandbox.stub(userActions, 'associateUserToOrg').returns(Promise.resolve());
+      sandbox.stub(AppDispatcher, 'handleServerAction');
+      userActions.receiveUserForCF(user, inviteData)
+        .then(done, done.fail);
+    });
+
+    it('confirms the user was created and returns a guid', function () {
+      expect(AppDispatcher.handleServerAction).toHaveBeenCalledWith(sinon.match({
+        type: userActionTypes.USER_IN_CF_CREATED,
+        user
+      }));
+    });
+
+    it('should send off email invite with invite data', function () {
+      expect(userActions.sendUserInviteEmail).toHaveBeenCalledOnce();
+    });
+
+    it('should call next request to associate user', function () {
+      expect(userActions.associateUserToOrg).toHaveBeenCalledOnce();
+    });
+  });
+
+  describe('sendUserInviteEmail', function () {
+    let userGuid;
+    let inviteData;
+    let spy;
+    beforeEach(function () {
+      userGuid = "fake-user-udid";
+      inviteData = { new_invites: [{ userId: userGuid }] };
+      spy = setupServerSpy(sandbox);
+
+      sandbox.stub(uaaApi, 'sendInviteEmail');
+      userActions.sendUserInviteEmail(inviteData);
+    });
+
+    it('should announce email invite to actions', function () {
+      assertAction(spy, userActionTypes.USER_EMAIL_INVITE);
+    });
+
+    it('should trigger request for invite email', function () {
+      expect(uaaApi.sendInviteEmail).toHaveBeenCalledOnce();
+    });
+  });
+
+  describe('associateUserToOrg', function () {
+    let user;
+    let userGuid;
+    let orgGuid;
+    beforeEach(function (done) {
+      userGuid = "fake-udid";
+      orgGuid = "fake-org-guid";
+      user = { guid: userGuid };
+      sandbox.stub(OrgStore, 'get').returns(orgGuid);
+      sandbox.stub(cfApi, 'putAssociateUserToOrganization').returns(Promise.resolve());
+      sandbox.stub(userActions, 'associatedUserToOrg').returns(Promise.resolve());
+      sandbox.stub(AppDispatcher, 'handleServerAction');
+      userActions.associateUserToOrg(user)
+        .then(done, done.fail);
+    });
+
+    it('completes the association to user and org', function () {
+      expect(AppDispatcher.handleServerAction).toHaveBeenCalledWith(sinon.match({
+        type: userActionTypes.USER_ORG_ASSOCIATION_FETCH
+      }));
+    });
+
+    it('should trigger cf api to make put request to associate user', function () {
+      expect(cfApi.putAssociateUserToOrganization).toHaveBeenCalledOnce();
+    });
+
+    it('should call associatedUser confirmation after', function () {
+      expect(userActions.associatedUserToOrg).toHaveBeenCalledOnce();
+    });
+  });
+
+  describe('associatedUserToOrg', function () {
+    it('should dispatch a view event with user and org associated', function() {
```

Comment:
just change this language since the `associatedUserToOrg` itself doesn't dispatch a `view event`. it just dispatches the `USER_ORG_ASSOCIATION_RECEIVED ` action. it'll be up to the org store when we register it in another PR when the view will actually change